### PR TITLE
[#651] Use security secrets to serialize ActiveMQArtemisSecurity CRs

### LINKED ISSUE #651: ActiveMQSecurity CR causes error argument list too long
ActiveMQSecurity CRs with many securitySettings cause the following error:
```
exec /bin/bash: argument list too long
```

### PR DESCRIPTION
**Pull request by brusdev**

### CODE REVIEW

File: controllers/activemqartemissecurity_controller.go [hunk 1]
```diff
@@ -252,15 +266,11 @@ func (r *ActiveMQArtemisSecurityConfigHandler) getPassword(secretName string, ke
 
 func (r *ActiveMQArtemisSecurityConfigHandler) Config(initContainers []corev1.Container, outputDirRoot string, yacfgProfileVersion string, yacfgProfileName string) (value []string) {
 	ctrl.Log.Info("Reconciling ActiveMQArtemisSecurity", "cr", r.SecurityCR)
-	result := r.processCrPasswords()
 	outputDir := outputDirRoot + "/security"
 	var configCmds = []string{"echo \"making dir " + outputDir + "\"", "mkdir -p " + outputDir}
 	filePath := outputDir + "/security-config.yaml"
-	cmdPersistCRAsYaml, err := r.persistCR(filePath, result)
```
Comment (gaohoward) [inline controllers/activemqartemissecurity_controller.go:259]: The persistCR() method is not used any more, it can be deleted too.

**Review by gaohoward — CHANGES_REQUESTED** [review]

File: controllers/activemqartemissecurity_controller.go (hunk 1 above)
Comment (brusdev) [reply controllers/activemqartemissecurity_controller.go:259]: done

**Comment by gaohoward** [comment]
LGTM

**Review by gtully — APPROVED** [review]
you are able to use the existing secret, that is great.

**Pull request merged**